tools/accuracy_checker/configs/single-human-pose-estimation-0001.yml

```
@@ -0,0 +1,21 @@
+models:
+  - name: single-human-pose-estimation-0001
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        model:   intel/single-human-pose-estimation-0001/FP32/single-human-pose-estimation.xml
+        weights: intel/single-human-pose-estimation-0001/FP32/single-human-pose-estimation.bin
```

```suggestion
        weights: intel/single-human-pose-estimation-0001/FP32/single-human-pose-estimation-0001.bin
```
other intel models have consistent naming with folder

---

Will this model come from `openvino_trainng_extensions`? If so, it should go to `public` folder.

---

Yes, you are right. Thank you.